vars/test.yml

```diff
@@ -185,6 +185,9 @@ secureboot_enabled: false
 memory_mb: 4096
 cpu_number: 2
 cpu_cores_per_socket: 2
+# Whether update vm's memory and cpu when 'vm_deploy_method' is set to 'ova': default is false.
+# If it's true, vm will be re-configured with 'memory_mb', 'cpu_number' and 'cpu_cores_per_socket' settings.
+update_vm_mem_cpu: false
```

Comment:
I think this parameter can be moved down after `ova_path` as there is comment above `Set below parameters when 'new_vm' is true and 'vm_deploy_method' is set to 'iso'`, and the parameters following it are for ISO VM.